test/end-to-end/core.sh

```diff
@@ -464,7 +464,6 @@ echo "[INFO] Validating routed app response..."
 # will be reachable via the ip of its pod.
 router_ip=$(oc get pod "${router_pod}" --template='{{.status.podIP}}')
 CONTAINER_ACCESSIBLE_API_HOST="${CONTAINER_ACCESSIBLE_API_HOST:-${router_ip}}"
-validate_response "-s -k --resolve www.example.com:443:${CONTAINER_ACCESSIBLE_API_HOST} https://www.example.com" "Hello from OpenShift" 0.2 50
```

Comment:
did the map change break the "default cert" behavior of the router? I really don't want to remove this test if it was previously passing and suddenly failed with this PR